api/dp-openapi.yaml

```diff
@@ -591,6 +591,350 @@ paths:
         default:
           $ref: '#/components/responses/UnexpectedError'
 
+  /projects/badpagedata:
+    get:
+      tags:
+      - project
+      description: Gets data for mark bad page
+      responses:
+        200:
+          description: Data
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/bad_page_data'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '429':
+          $ref: '#/components/responses/RateLimitExceeded'
+        default:
+          $ref: '#/components/responses/UnexpectedError'
+
+  /projects/{projectid}/pickersets:
+    get:
+      tags:
+      - project
+      description: Gets pickersets for the project
+      parameters:
+      - name: projectid
+        in: path
+        description: ID of project
+        required: true
+        schema:
+          type: string
+      responses:
+        200:
+          description: pickersets
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/pickersets'
+        401:
+          $ref: '#/components/responses/Unauthorized'
+        404:
+          $ref: '#/components/responses/NotFound'
+        429:
+          $ref: '#/components/responses/RateLimitExceeded'
+        default:
+          $ref: '#/components/responses/UnexpectedError'
+
+  /rounds/{roundid}:
+    get:
+      tags:
+      - project
+      description: Gets round information
+      parameters:
+      - name: roundid
+        in: path
+        description: The round identifier (as defined by array_keys($Round_for_round_id_))
```

Comment:
These descriptions should not reference PHP code. They should be useful to users without seeing the code.

Reply:
If this is linked from existing code these rounds are provided by the links but in general I supose we should add an endpoint to get a list of valid rounds for proofreading in the same way as projects/pagerounds but noit including OCR.